spec/features/trainer_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+require 'support/helpers/trainer_helper.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.